src/flows/components/panel/Handle.tsx

```diff
@@ -0,0 +1,55 @@
+// Libraries
+import React, {FC, RefObject} from 'react'
+import classnames from 'classnames'
+
+interface Props {
+  testID?: string
+  dragRef: RefObject<HTMLDivElement>
+  onStartDrag: () => void
+  dragging: boolean
+}
+
+const Handle: FC<Props> = ({
+  testID = 'notebook-resize-handle',
+  dragging = false,
+  onStartDrag = () => {},
+  dragRef,
+}) => {
+  const handleMouseDown = (): void => {
+    onStartDrag()
+  }
+
+  const DraggableResizerHandleClass = classnames(
+    'cf-draggable-resizer--handle',
+    {
+      'cf-draggable-resizer--handle-dragging': dragging,
+    }
+  )
+
+  return (
+    <div
+      className={DraggableResizerHandleClass}
+      onMouseDown={handleMouseDown}
+      title="Drag to resize"
+      data-testid={testID}
+      ref={dragRef}
+    >
+      <div className="cf-draggable-resizer--handle-pill1 cf-draggable-resizer--handle-pill1--horizontal"></div>
+      <div
+        className="cf-draggable-resizer--gradient1 cf-draggable-resizer--gradient1--horizontal"
+        style={{
+          background: `linear-gradient(to right, #ce58eb 0%, #00C9FF 100%)`,
```

Comment:
why can't this be achieved in the `scss` file?

Reply:
should be handled in clockface, but it isn't. so this is a stand in while we try to talk about exposing the drag handle in that library. wanted to have the least amount of impact so i can just delete a single file when that time comes

Reply:
Sounds good. Is there a clockface issue to track this to get it out?